src/js/themes/__tests__/__snapshots__/theme-test.js.snap

```diff
@@ -1658,7 +1658,7 @@ exports[`Grommet hpe theme 1`] = `
   display: flex;
   box-sizing: border-box;
   max-width: 100%;
-  background-color: #F8F8F8;
+  background-color: #00E8CF;
```

Comment:
seem right? Related to updates to the HPE NEXT theme?

Reply:
I don't think so, the test is using hpe v1.0.0 and not the NEXT theme, so I'm not sure why this change occurs. we didn't have hpe-theme changes or releases in the last few weeks.